python/pyspark/worker.py

```diff
@@ -343,6 +377,30 @@ def map_batch(batch):
         # profiling is not supported for UDF
         return func, None, ser, ser
 
+    # Helper function to extract the key and value indexs from arg_offsets
```

Comment:
The doc here gives a good description, but could you make it clear it's only for group udfs? Also, please change it to the normal pydoc style for a function.